### PR TITLE
[LM-12816] Remaining bug fixes

### PR DESCRIPTION
**Pull request by divyanshgandhilm**
## Description
Fixing all remaining bugs from previous releases

**Features**

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [ ] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: lib/packages/core/lib/src/views/media/media_preview.dart [hunk 1]
```diff
@@ -120,45 +134,82 @@ class _LMChatMediaPreviewScreenState extends State<LMChatMediaPreviewScreen> {
...
+      subtitle: _buildAppBarSubtitle(),
     );
   }
 
+  Widget _buildAppBarSubtitle() {
+    return ValueListenableBuilder(
+      valueListenable: _currentPosition,
+      builder: (context, position, _) {
+        final DateTime now = DateTime.now();
+        final String formattedDate =
```
Comment (AnujLM) [inline lib/packages/core/lib/src/views/media/media_preview.dart:146]: can we use DateFomate from itl package here?

File: lib/packages/core/lib/src/views/media/media_preview.dart [hunk 2]
```diff
@@ -120,45 +134,82 @@ class _LMChatMediaPreviewScreenState extends State<LMChatMediaPreviewScreen> {
...
-          mediaList.first.mediaType == LMChatMediaType.video) {
+    if (mediaList.isEmpty) {
+      return const SizedBox();
+    }
+
+    return CarouselSlider.builder(
+      carouselController: _carouselController,
+      itemCount: mediaList.length,
+      options: CarouselOptions(
+        height: MediaQuery.of(context).size.height,
```
Comment (AnujLM) [inline lib/packages/core/lib/src/views/media/media_preview.dart:188]: user `MediaQuery.sizeOf(context).height` for better performance.

File: lib/packages/ui/lib/src/widgets/media/document/preview.dart [hunk 3]
```diff
@@ -77,47 +76,48 @@ class _LMChatDocumentPreviewState extends State<LMChatDocumentPreview> {
...
-                                ),
-                              ),
-                            ],
-                          )
-                        ],
-                      ),
-                    ),
-                  ],
-                ),
+                //   Column(
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/document/preview.dart:79]: remove commented code, if not requried.

**Review by AnujLM — CHANGES_REQUESTED** [review]

**Comment by divyanshgandhilm** [comment]
Fixed these issues in the next PR

**Pull request merged**